### DISCUSSION TITLE
How to use Security Onion

### DISCUSSION THREAD

**Discussion by nicholaswkc34** [Security-Onion-Solutions/securityonion#9814] [Unsupported Versions]
Dear forumer, 
May i know how to use security onion??? Which router to buy and how to design the network?

**Comment by Acewiza** [comment]
A little intro and background would be helpful for such a wide-ranging question.

**Comment by nicholaswkc34** [comment]
I have opnsense router as gateway connect to moderm.

**Comment by nicholaswkc34** [comment]
Modem - OPNSense Gateway -> Linksys Router

**Comment by Jaap79** [comment]
Considering your question, I would ask myself the following:

- What is your intended purpose for SecurityOnion?
- Do you have servers to spare, or enough resources (CPU, memory, storage) for a SecurityOnion deployment?
- What is it that you want to monitor, and is SecurityOnion the way to go?

Is it purely for home use? Then go for a single node deployment. Get a decent server able to capture your network traffic and read up on an EVAL deployment for instance. Try it out and see what it does/if it works for you. 

But without this basis nobody here can recommend what you should or should not build. If you defined a purpose for it, we might be able to help you out with specific details..

If you're wondering whether you have to buy a specific router to use SecurityOnion, it could happen that SecurityOnion is not be the product you're looking for. SO doesn't care about your modem or router (except maybe when outputting Syslog data.. but that's a different story and something you won't have to worry about for a while).

**Comment by nicholaswkc34** [comment]
I want to use Security Onion to monitor my network. I have allocate the required hardware for SO.

**Reply by Acewiza** [reply]
Sorry, no experience with Opensense and Linksys.  In any event, I suspect you have alot of reading to do before posing questions here.  They would need to be much more specific, like addressing configuration and installation issues, etc.

**Comment by Sazerak** [comment]
As others have mentioned, this is really too vague of a question for anyone to help. If you want to know how to "use" Security Onion, start by reading the documentation: https://docs.securityonion.net/en/2.3/index.html

If and when you run into a specific question/issue, that's the time to be posting a question (after searching to see if it's already been answered of course)
